src/components/DataTable/examples/07.empty-with-custom-title-and-body.jsx

```diff
@@ -15,7 +15,7 @@ export default createClass({
 		const { data } = this.state;
 
 		return (
-			<DataTable data={data} density="extended" isFullWidth>
+			<DataTable data={data} density="extended" isFullWidth minRows={15}>
```

Comment:
Why have this example show a custom `minRows` when there's another example that also has custom `minRows`?